wordpress-mobile/WordPress-iOS
# Jetpack Onboarding Prompt: Style the prompt view

Project: #17721 

### Description
Adds the initial style and copy for the Jetpack install prompt view 

**Notes:** 
- It is expected that the UI tests are broken for this PR. Testing will be addressed in a later PR.
- The copy isn't final yet


### Screenshots
| iPhone | iPad |
|:---:|:---:|
|![Simulator Screen Shot - iPhone 11 Pro - 2022-01-12 at 10 20 39](https://user-images.githubusercontent.com/793774/149191759-f190ccdb-bb5c-4af3-8a89-547a0971c42e.png)|![Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2022-01-12 at 10 59 14](https://user-images.githubusercontent.com/793774/149191756-60be4159-257c-4e85-98cf-ca70a7da7e43.png)|

### Video
https://user-images.githubusercontent.com/793774/149192986-bcc161c5-42a1-47b2-bd04-00aed335678d.mp4

### To test:

1. Launch the app
2. Logout
3. Sign in to a site that is not connected to Jetpack
4. Verify you are shown the prompt view as displayed above
5. Tap on the Learn more button
6. Verify a webview opens and you are brought to the Jetpack features page
7. Tap the 'Install' button and verify the install is presented


### Regression Notes
1. Potential unintended areas of impact
None. 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

3. What automated tests I added (or what prevented me from doing so)
None yet. 

### PR submission checklist:
